CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# Contributing to Common Voice
+
+🎉 First off, thanks for taking the time to contribute! This project would not be possible without people like you. 🎉
+
+## Code of Conduct
+By participating in this project, you're agreeing to uphold the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
+
+## How Can I Contribute?
+
+### Non-code
```

Comment:
I think we should rephrase this section in a more positive way. (Though we can do this in a subsequent PR.)

"Non-code" can easily be taken to mean "so you're not techy/nerdy/smart/<other adjectives> enough to contribute code?"

Since extending the sentence corpus is actually a huge part of CV, I'd go with something like "Add more sentences" or "Contribute to the text corpus" for this. :)

Reply:
@nevik  This was _exactly_ what I meant by the headlines need some help! I agree with everything you said, I just wasn't sure what a suitable alternative would be. I'll let you and @mikehenrty make the final call.